lms/services/organization.py

```diff
@@ -383,6 +384,24 @@ def get_hierarchy_ids(self, id_, include_parents=False) -> list[int]:
         rows = self._db_session.query(base_case.union(recursive_case)).all()  # type: ignore
         return [row[0] for row in rows]
 
+    def is_member(self, organization: Organization, user: User) -> bool:
+        """
+        Check if user is a member organization.
+
+        We define "member" here as belonging to any of the organization's courses.
+        """
+        query = (
+            select(GroupingMembership)
+            .join(Course)
+            .join(ApplicationInstance)
+            .where(
+                GroupingMembership.user_id == user.id,
+                ApplicationInstance.organization_id == organization.id,
+            )
+            .limit(1)
+        )
+        return bool(self._db_session.execute(query).scalar())
+
```

Comment:
Perhaps a micro-optimization, but if we just want to know if there's a result for that query, maybe we can do a `SELECT id ...`, or something in those lines, so that the whole model doesn't need to be mapped.

Although, maybe `execute(query).scalar()` does precisely that, so don't give me too much credit 😅 

Or, if you think it's just not worth the effort, I'm also ok with that.

Reply:
Yeah, that's a good point although in the grand scheme of things, how the LMS app works, I don't think is worth the effort.